src/service/viewer-impl.js

```diff
@@ -79,7 +78,17 @@ export const VisibilityState = {
   /**
    * Viewer has indicated that AMP document is hidden.
    */
-  HIDDEN: 'hidden'
+  HIDDEN: 'hidden',
+
+  /**
+   * Viewer is in the middle of a swipe, so the AMP document is "paused"
+   */
+  PAUSED: 'paused',
+
+  /**
+   * TODO
+   */
+  UNLOADED: 'unloaded',
```

Comment:
I think a better term for this might be "background". This state only triggers when the browser tab that displays the viewer/doc is not active (another tab is).